packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.styles.ts

```diff
@@ -212,11 +212,18 @@ export const getStyles = (props: IChoiceGroupOptionStyleProps): IChoiceGroupOpti
         selectors: {
           '.ms-Label': {
             fontSize: FontSizes.medium,
-            padding: '0 0 0 26px',
```

Comment:
Why are you deleting the default 0 out.  Shouldn't this be `padding: 0;`  Notice in the snapshot that the padding: 0 is removed.  I imagine removing that 0 out could cause some breaking style changes

Reply:
I'm not sure why those were set in the first place, as there is no padding by default that needs to be unset. There isn't even an actual Label component that would bring padding of its own, this is just a span.

Reply:
Ok if there's no regressions.